### PR TITLE
Add support for enabling or disabling SSLCert verification

### LINKED ISSUE #12: [Feature] Add support for disabling TLS/SSL verification
Currently there's no way to disable TLS/SSL verification when using this library. 

Example:
```
import requests
from urllib3.exceptions import InsecureRequestWarning

# Suppress only the single warning from urllib3 needed.
requests.packages.urllib3.disable_warnings(category=InsecureRequestWarning)

# Set `verify=False` on `requests.post`.
requests.post(url='https://example.com', verify=False)
```

### PR DESCRIPTION
**Pull request by gaby**
* Add support for dependabot to manage pip/poetry dependencies
* Add support for `verify` to Enable/Disable SSL Cert verification for `requests` (Defaults to True)
* Bump  `poetry` to 1.1.14
* Bump `requests` to 2.28.1

Source for `urllib3` warning was taken from here: https://stackoverflow.com/a/28002687

Closes #12

### CODE REVIEW

**Comment by gaby** [comment]
@PetrS12 We ran into issues with SSLCert verification while using `rabbitmq-api-admin` in a test environment, since the certs are temporary.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
     container: python:${{ matrix.python }}
     strategy:
       matrix:
-        python: [3.8, 3.9, 3.10.0]
+        python: [3.8, 3.9, 3.10]
```
Comment (PetrS12) [inline .github/workflows/tests.yml]: it`s not just how it was done;) Zero is discarded with this type of record "3.10"

Comment (gaby) [reply .github/workflows/tests.yml]: Ohhh, I will add it back.

Comment (gaby) [reply .github/workflows/tests.yml]: Done

**Comment by gaby** [comment]
@PetrS12 I just sync the PR with master, and added `3.10.0` back into the tests yml.

**Review by PetrS12 — APPROVED** [review]

**Pull request merged**